FusionIIIT/applications/academic_procedures/admin.py

```diff
@@ -1,10 +1,10 @@
 from django.contrib import admin
 
-from .models import (BranchChange, CoursesMtech, InitialRegistrations,
-                     MinimumCredits, Register, Thesis,
+from .models import (Bonafide, BranchChange, CourseRequested, CoursesMtech, FeePayments, FinalRegistration, InitialRegistration, InitialRegistrations,
```

Comment:
Ahh, you just missed registering, the Bonafide model. 
Register that and we are good to go!

Reply:
I have knowingly removed Bonafide table from admin.py
The admin dashboard breaks while opening bonafide table due to some db error.
This can handled as a separate issue itself

Reply:
Ohk Garvit. Approving this PR.